src/Microsoft.OpenApi/Models/OpenApiResponse.cs

```diff
@@ -136,7 +140,7 @@ public void SerializeAsV2(IOpenApiWriter writer)
 
                         foreach (var example in Content
                             .Where(mediaTypePair => mediaTypePair.Value.Examples != null && mediaTypePair.Value.Examples.Any())
-                            .SelectMany(mediaTypePair => mediaTypePair.Value.Examples))
+                            .SelectMany(mediaTypePair => mediaTypePair.Value.Examples!))
```

Comment:
don't use bang operators , they usually lead to bugs.
Add an OfType<Foo>() after the select many instead, not only it'll filter the null values at runtime, it'll fix the compiler complaining about nullable values.

Reply:
Using OfType() still doesn't work. There's still a possible null reference return at the SelectMany():
```c#
foreach (var example in Content
    .Where(mediaTypePair => mediaTypePair.Value.Examples is not null && mediaTypePair.Value.Examples.Any())
    .SelectMany(mediaTypePair => mediaTypePair.Value.Examples)
    .OfType<KeyValuePair<string, IOpenApiExample>>())
{
    writer.WritePropertyName(example.Key);
    example.Value.SerializeAsV2(writer);
}
```

Reply:
See b6d68bbb430b5e08d701f6a831fe07245dbecad8

Reply:
Thank you!